changelogs/unreleased/66722.yml

```diff
@@ -0,0 +1,5 @@
+title: "fix stock details by product productionIndicators visibility"
```

Comment:
Change title to add the 'object' affected, can be like this "Stock details: fix stock details ..."